include/openssl/tls1.h

```diff
@@ -868,6 +872,9 @@ SSL_CTX_callback_ctrl(ssl,SSL_CTRL_SET_TLSEXT_TICKET_KEY_CB,(void (*)(void))cb)
 # define TLS1_TXT_DHE_PSK_WITH_CHACHA20_POLY1305           "DHE-PSK-CHACHA20-POLY1305"
 # define TLS1_TXT_RSA_PSK_WITH_CHACHA20_POLY1305           "RSA-PSK-CHACHA20-POLY1305"
 
+/* TLSv1.3 ciphersuites */
+# define TLS1_3_TXT_AES_128_GCM_SHA256                     "TLS_AES_128_GCM_SHA256"
```

Comment:
The number of people who are familiar with OpenSSL names greatly exceeds the few-dozen people who are familiar with the "official" RFC names.

Before we go too far worry about names, I agree with @davidben that we should decouple the cipher spec from the current CIPHERS variable.